utils/e2e-tests/ts/lib/abis/contractsFactory.ts

```diff
@@ -0,0 +1,66 @@
+// pragma solidity ^0.8.13;
+//
+// contract ContractsFactory {
+//     address private owner;
+//
+//     /**
+//      * Sets contract deployer as owner.
+//      */
+//     constructor() payable {
+//         owner = msg.sender; // 'msg.sender' is sender of current call, contract deployer for a constructor
+//     }
+//
+//     function build()
+//         public
+//         returns (Item itemAddress)
+//     {
+//         return new Item();
+//     }
+//
+//     function withdrawAll() external isOwner {
+//         (bool success,) = owner.call{value:address(this).balance}("");
+//         require(success, "Transfer failed!");
+//     }
+//
+//     /**
+//      * Makes sure if caller is owner.
+//      */
+//     modifier isOwner() {
```

Comment:
Do we need this check? I think we should simplify the test contract, and this here seems like something that the trst can work fine without

Reply:
I agree, simplified